server/views/resources/links.jade

```diff
@@ -2,57 +2,145 @@ extends ../layout
+                                td
+                                    a(href="//github.com/freecodecamp", target='_blank') GitHub Repository
+                            tr
+                                td.text-right
+                                    .ion-social-reddit
+                                td
+                                    a(href="//www.reddit.com/r/freecodecamp", target='_blank') Subreddit
+                            tr
+                                td.text-right
+                                    .ion-social-linkedin
+                                td
+                                    a(href="//www.linkedin.com/edu/school?id=166029", target='_blank') LinkedIn University Page
+                            tr
+                                td.text-right
+                                    .ion-social-twitter
+                                td
+                                    a(href="//twitter.com/freecodecamp", target='_blank') Twitter Feed
+                            tr
+                                td.text-right
+                                    .ion-social-facebook
+                                td
+                                    a(href="//facebook.com/freecodecamp") Facebook Page
+                            tr
+                                td.text-right
+                                    .ion-social-twitch-outline
+                                td
+                                    a(href="//twitch.tv/freecodecamp", target='_blank') Twitch.tv Channel
+                        .spacer
+
+                        h2 Other useful links:
+                        table.table.link-table
+                            tr
+                                td.text-right
+                                    .ion-locked
+                                td
+                                    a(href="/terms-and-privacy") Our Privacy Policy, Terms of Service and Code of Conduct
+                            tr
+                                td.text-right
+                                    .ion-erlenmeyer-flask
+                                td
+                                    a(href="/labs") Extra-curricular Apps Built by Campers
+                            tr
+                                td.text-right
+                                    .ion-chatbox
+                                td
+                                    a(href="/stories") Stories from campers who've become professional developers
+                        .spacer
+                        h2 Which camper to contact for:
+                        table.table.table-stripe
+                            tr
+                                td Support (team@freecodecamp.com)
+                                td
+                                    a(href='https://gitter.im/QuincyLarson' target='_blank') @QuincyLarson
+                            tr
+                                td Facebook page
```

Comment:
I think all of these should be in Title case:
Facebook Page
Twitter Feed
etc.

Some are in title case, some are not.